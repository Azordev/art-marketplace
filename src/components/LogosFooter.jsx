import React from "react";
import LogoFooter from "../assets/img/logo.png";
import LogoMinisterio from "../assets/img/logo-2.png";
import LogoBicentenario from "../assets/img/logo-3.png";

const LogosFooter = () => (
  <section className="max-w-screen-2xl mx-auto bg-white px-5 lg:px-10 2xl:px-20 pb-10">
    <div className="flex flex-col items-center justify-center space-y-8 lg:space-y-0 lg:space-x-4 xl:space-x-0 lg:flex-row gap-12">
      <div>
        <img
          src={LogoMinisterio}
          alt=""
          className="block object-scale-down lg:w-80 lg:h36"
        />
      </div>
      <div>
        <img
          src={LogoFooter}
          alt=""
          className="block object-scale-down lg:w-80 lg:h36"
        />
      </div>
      <div>
        <img
          src={LogoBicentenario}
          alt=""
          className="block object-scale-down lg:w-80 lg:h36"
        />
      </div>
    </div>
  </section>
);

export default LogosFooter;
