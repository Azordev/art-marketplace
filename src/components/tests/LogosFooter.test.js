import React from 'react';
import renderer from 'react-test-renderer';
import LogosFooter from '../LogosFooter';

it('renders correctly', () => {
  const tree = renderer
    .create(<LogosFooter />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
