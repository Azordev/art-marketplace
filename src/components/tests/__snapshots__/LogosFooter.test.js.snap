// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="max-w-screen-2xl mx-auto bg-white px-5 lg:px-10 2xl:px-20 pb-10"
>
  <div
    className="flex flex-col items-center justify-center space-y-8 lg:space-y-0 lg:space-x-4 xl:space-x-0 lg:flex-row gap-12"
  >
    <div>
      <img
        alt=""
        className="block object-scale-down lg:w-80 lg:h36"
        src="logo-2.png"
      />
    </div>
    <div>
      <img
        alt=""
        className="block object-scale-down lg:w-80 lg:h36"
        src="logo.png"
      />
    </div>
    <div>
      <img
        alt=""
        className="block object-scale-down lg:w-80 lg:h36"
        src="logo-3.png"
      />
    </div>
  </div>
</section>
`;
